exec/java-exec/src/main/java/org/apache/drill/exec/ops/FragmentContext.java

```diff
@@ -392,6 +402,11 @@ public String getQueryUserName() {
     return fragment.getCredentials().getUserName();
   }
 
+  /**
+   * @return ID {@link java.util.UUID} of the current query
+   */
+  public String getQueryId() { return QueryIdHelper.getQueryId(fragment.getHandle().getQueryId());}
```

Comment:
We use the term "query ID" for the numeric version of the query ID as shown by the `getHandle().getQueryId()` method. Suggestion: rename this to 'getQueryIdString`. And, if we do that, provide a version that returns the original query ID. That is:
```
public QueryId getQueryId() { return fragment.getHandle().getQueryId(); }
public String getQueryIdString() { return QueryIdHelper.getQueryId(getQueryId()); }
```